docs/source-app/glossary/secrets.rst

```diff
@@ -0,0 +1,57 @@
+.. _secrets:
+
+#################
+Encrypted Secrets
+#################
+
+We understand that many Apps require access to private data like API keys, access tokens, database passwords, or other credentials. And that you need to protect this data.
```

Comment:
I would remove "We understand", just say "Many apps require access..."

Reply:
@Felonious-Spellfire

Reply:
Please review my PR. The structure was completely changed